android/src/main/kotlin/com/onegini/mobile/sdk/flutter/useCases/ResourceRequestUseCase.kt

```diff
@@ -49,12 +51,14 @@ class ResourceRequestUseCase @Inject constructor(private val oneginiSDK: Onegini
   }
 
   private fun getCompleteResourceUrl(path: String): String {
-    // TODO Add support for multiple base resource urls; https://onewelcome.atlassian.net/browse/FP-38
     val resourceBaseUrl = oneginiSDK.oneginiClient.configModel.resourceBaseUrl
 
-    return when (path.startsWith(resourceBaseUrl)) {
-      true -> path
-      else -> resourceBaseUrl + path
+    // Check for absolute or relative url
```

Comment:
Is this Correct? What happens if your resourceBaseUrl + Path is not a valid url?

Reply:
then the request will fail, with a 404 or 400 which is intentional

Reply:
hmm maybe you are right lemme test

Reply:
fixed